CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The changes are relative to the previous release, unless the baseline is specifi
 
 ## [Unreleased]
 
+## [1.2.0] - 2025-02-13
```

Comment:
Remove "Also add Unreleased section in CHANGELOG.md." from the commit message, because the Unreleased section should not be removed in https://github.com/AOMediaCodec/libavif/pull/2614.

The net change in this pull request (after https://github.com/AOMediaCodec/libavif/pull/2614 is merged) should be changing `AVIF_VERSION_DEVEL` from 0 to ` in avif.h. See https://github.com/AOMediaCodec/libavif/commit/175626239f501902b7eb6fccc81f971a58ae6fca for an example.

Reply:
Done. That was on purpose.